static/app/components/onboarding/gettingStartedDoc/utils/replayOnboarding.tsx

```diff
@@ -3,6 +3,15 @@ import type {DocsParams} from 'sentry/components/onboarding/gettingStartedDoc/ty
 import {tct} from 'sentry/locale';
 
 export const getReplayConfigureDescription = ({link}: {link: string}) =>
+  tct(
+    'The SDK is aggressively redacting all text and images. We plan to add fine controls for redacting, but currently, we just allow either on or off. Learn more about configuring Session Replay by reading the [link:configuration docs].',
+    {
+      code: <code />,
+      link: <ExternalLink href={link} />,
+    }
+  );
+
+export const getReplayMobileConfigureDescription = ({link}: {link: string}) =>
```

Comment:
are the bodies of these function swapped?

Reply:
oops yes! fixed